__tests__/app.test.js

```diff
@@ -14,27 +14,37 @@ afterAll(() => {
 });
 
 describe("/api/topics", () => {
-  test("GET:200 sends an array of topics", () => {
+  test("GET:200 responds with an array of topics", () => {
     return request(app)
       .get("/api/topics")
       .expect(200)
       .then((response) => {
-        expect(response.body.topics.length).toBe(3)
+        expect(response.body.topics.length).toBe(3);
         response.body.topics.forEach((topic) => {
           expect(topic).toHaveProperty("slug");
           expect(topic).toHaveProperty("description");
         });
       });
   });
 
-  test("GET:400 sends an appropriate message when given an invalid path", () => {
+  test("GET:400 responds with an appropriate message when given an invalid path", () => {
     return request(app)
       .get("/api/topicnotatopic")
       .expect(400)
-      .then((response) => { 
-          
-        expect(response.body.msg).toBe('Bad Request')
+      .then((response) => {
+        expect(response.body.msg).toBe("Bad Request");
+      });
+  });
+});
+
+describe("/api", () => {
+  test("GET:200 provides a description of all other endpoints available", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((response) => {
        
-        });
+        expect(response.body.endPoints).toEqual(endPoints)
       });
   });
```

Comment:
This test is good, requiring in the json and comparing against the response body is good :)